paasta_tools/chronos_tools.py

```diff
@@ -185,6 +186,13 @@ def __eq__(self, other):
             (self.branch_dict == other.branch_dict)
         )
 
+    def __normalize(self):
+        """When the config supports alternative syntaxes, convert them
+        to a normal form."""
+        parents = self.get_parents()
+        if parents is not None and not isinstance(parents, list):
+            self.config_dict['parents'] = [parents]
```

Comment:
It seems a little strange to me that it mutates the config_dict.

What if you moved this code to the `get_parents` getter?